examples/meta/src/evaluation/multiclassovrevaluation.sg.in

```diff
@@ -0,0 +1,29 @@
+File traindat = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+File label_traindat = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Labels ground_truth_labels = labels(label_traindat)
+#![create_features]
+
+#![create_classifier]
+Machine svm= machine("MulticlassLibLinear", C=1.0, labels=ground_truth_labels, seed=1)
+Parallel p = svm.get_global_parallel()
+p.set_num_threads(1)
```

Comment:
we dont want this in the example

Reply:
you mean the parallel part?

Reply:
yes! I am not sure that breaks the integration test, but if it does we better fix it rather than having thick hack here

Reply:
Got it! I have removed the two lines but the problem comes from `accuracy_multiclass` It fails the integration tests by a segfault any ideas what could be the cause?

Reply:
Fixed it!

Reply:
what was the problem? I don't see a fix here ....

Reply:
@karlnapf the problem was the data file I made itself, it wasn't written properly so it gave a segfault error on integration, makes sense why only the integration tests were failing, anyways I have made a PR in the shogun-data, plus commited it here aswell but since I have already changed the file it made a conflict as you can see in the CI

Reply:
you wrote the file yourself? :laughing:  why?
Anyways, glad you figured out the problem.! :)
You need to rebase and then we can merge once the CI is happy

Reply:
CI is happy :P and how am I supposed to generate the .dat file

Reply:
Check the readme. Please *DO NOT* submit PRs with .dat files that you created yourself by hand (the CI will also not accept those)